spec/features/user_index_page_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  before :each do
```

Comment:
- [ ] Kindly make sure all your tests are passing as required for the project. From the image below you can see that there are still **2 tests** which are not passing:
![Screenshot from 2023-01-31 09-15-38](https://user-images.githubusercontent.com/83042742/215692399-65319ea1-b7e1-44d3-9fa5-dc5975b123c8.png)